test/CMakeLists.txt

```diff
@@ -99,6 +119,18 @@ macro(_add_test _NAME _DEPENDS)
     add_dependencies(${_NAME} ${_NAME}_data)
     add_test(NAME ${_NAME} COMMAND ${_NAME})
     target_link_libraries(${_NAME} fdeep Threads::Threads doctest::doctest)
+    # GNU in linux doesn't support "-mbig-obj" option.
```

Comment:
"GNU in linux doesn't support "-mbig-obj" option"

You mean MinGW does not support it, right?

Reply:
Yes it does, but I didn't test it. I will test in linux these days.